### PR TITLE
Make release(-notes) a normal composite action

### PR DESCRIPTION
**Pull request by frederikrothenberger**
This removes the node scaffolding that was required to run the action.

Test run of the composite action: https://github.com/dfinity/internet-identity/actions/runs/4532626193/jobs/7984382416#step:8:47

### CODE REVIEW

File: .github/actions/release/run [hunk 1]
```diff
@@ -166,5 +166,3 @@ cat "$section_wasm_verification" >> "$body" && echo >> "$body" && rm "$section_w
 cat "$body"
 
 echo "notes-file=$body" >> "$GITHUB_OUTPUT"
-
-[ -n "${GITHUB_ENV:-}" ] && echo "RELEASE_ACTION_BODY_FILE=$body" >>"$GITHUB_ENV"
```
Comment (nmattia) [inline .github/actions/release/run:170]: What was this for?

**Review by nmattia — COMMENTED** [review]
Thanks! ~~For context, this action predates composite actions~~

EDIT: I think that's the reason actually: https://github.com/actions/runner/issues/1478

File: .github/actions/release/run (hunk 1 above)
Comment (frederikrothenberger) [reply .github/actions/release/run:170]: This used to set the filename as an env variable in Github (see [here](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#environment-files)). But we don't actually use it, so I removed it.

Comment (nmattia) [reply .github/actions/release/run:170]: Are there no other occurrences of `RELEASE_ACTION_BODY_FILE`?

**Review by nmattia — APPROVED** [review]

**Pull request merged**